src/main/java/io/lettuce/core/api/async/RedisListAsyncCommands.java

```diff
@@ -98,6 +115,19 @@
      */
     RedisFuture<Long> llen(K key);
 
+    /**
+     * Atomically returns and removes the first/last element (head/tail depending on the
+     * wherefrom argument) of the list stored at source, and pushes the element at the
+     * first/last element (head/tail depending on the whereto argument) of the list stored at destination.
+     *
+     * @param source the source key.
+     * @param destination the destination type: key.
+     * @param args command arguments to configure {@code SOURCE} and {@code DESTINATION} direction.
```

Comment:
`{@code …}` refers to the method args. Mind the letter casing. I guess that applies to all other variants of `lmove` and `blmove`.

Reply:
fixed